WordPress/Classes/ViewRelated/Stats/Insights/StatsTotalInsightsCell.swift

```diff
@@ -11,6 +11,50 @@ struct StatsTotalInsightsData {
     public static func followersCount(insightsStore: StatsInsightsStore) -> StatsTotalInsightsData {
         return StatsTotalInsightsData(count: insightsStore.getTotalFollowerCount())
     }
+
+    public static func createCommentsTotalInsightsData(periodStore: StatsPeriodStore) -> StatsTotalInsightsData {
+        let periodSummary = periodStore.getSummary()
+
+        guard let periodSummary = periodStore.getSummary() else {
+            return StatsTotalInsightsData(count: 0)
+        }
+
+        let splitSummaryTimeIntervalData = SiteStatsInsightsViewModel.splitStatsSummaryTimeIntervalData(periodSummary)
+        let sparklineData: [Int] = makeSparklineData(countKey: \StatsSummaryData.commentsCount, splitSummaryTimeIntervalData: splitSummaryTimeIntervalData)
+        let commentsData = SiteStatsInsightsViewModel.intervalData(periodSummary, summaryType: .comments)
+
+        return StatsTotalInsightsData(count: commentsData.count, difference: commentsData.difference, percentage: commentsData.percentage, sparklineData: sparklineData)
+    }
+
+    public static func createTotalInsightsData(periodStore: StatsPeriodStore, statsSummaryType: StatsSummaryType) -> StatsTotalInsightsData {
+        let periodSummary = periodStore.getSummary()
+
+        guard let periodSummary = periodStore.getSummary() else {
+            return StatsTotalInsightsData(count: 0)
+        }
+
+        let splitSummaryTimeIntervalData = SiteStatsInsightsViewModel.splitStatsSummaryTimeIntervalData(periodSummary)
+        let sparklineData: [Int] = makeSparklineData(countKey: \StatsSummaryData.commentsCount, splitSummaryTimeIntervalData: splitSummaryTimeIntervalData)
```

Comment:
I think this one should be `likesCount` as it's used to create the likes data?

Reply:
Or can we work it out based on the `statsSummaryType` passed in, then we could just call into this method from `createCommentsTotalInsightsData`?